samples/performance/hpp_pipeline_cache/README.md

```diff
@@ -0,0 +1,105 @@
+<!--
+- Copyright (c) 2022, The Khronos Group
+-
+- SPDX-License-Identifier: Apache-2.0
+-
+- Licensed under the Apache License, Version 2.0 the "License";
+- you may not use this file except in compliance with the License.
+- You may obtain a copy of the License at
+-
+-     http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-
+-->
+
+# HPP Pipeline Management
+
+A transcoded version of the performance sample [Pipeline Cache](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/samples/performance/pipeline_cache) that illustrates the usage of the C++ bindings of vulkan provided by vulkan.hpp.
```

Comment:
Apologies, have not reviewed other HPP samples before, did we agree to also duplicate tutorials? This could make it harder for them to keep synced and up to date

Reply:
I don't think, we explicitly agreed on that... but it seemed to be logical to also adjust the tutorial text to the transcoded code.
But you're right, it doubles the potential work on modifying those samples with a transcoded version. Although those samples seem to be rarely touched at all.

Reply:
We talked about this on the last call and agreed that it's fine to duplicate the readmes. Especially if they contain code that obviously looks different for the hpp based samples.